phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -719,9 +721,6 @@ private boolean isMatchingMutation(Mutation expected, Mutation actual) throws IO
     }
 
     private boolean isVerified(Mutation mutation) throws IOException {
-        if (mutation instanceof Delete) {
```

Comment:
can deletes be verified? They won't have an empty column, will they?

Reply:
I will change the signature of this method. It is used only for Put mutations.